include/Reader/readerir.h

```diff
@@ -1317,6 +1325,7 @@ class GenIR : public ReaderBase {
   uint32_t TargetPointerSizeInBits;
   const uint32_t UnmanagedAddressSpace = 0;
   const uint32_t ManagedAddressSpace = 1;
+  static uint32_t TailCallCount;
 };
```

Comment:
How are you going to reconcile a static field like `TaillCallCount` with threading? I agree we need these limit-counter type things but we should figure out how they are supposed to work.